code/game/turfs/simulated/floor/reinf_floor.dm

```diff
@@ -25,18 +25,21 @@
 		..()
 	return //unplateable
 
-/turf/open/floor/engine/attackby(obj/item/C, mob/user, params)
-	if(!C || !user)
+/turf/open/floor/engine/wrench_act(mob/living/user, obj/item/wrench/W)
```

Comment:
iirc part of the reason for switching to wrench_act was so you wouldn't have to typecast solely to obj/item/wrench, that way you could create multitools with different paths that can call wrench_act too

usesound & toolspeed are obj/item level vars, so you should be fine to drop it down a level.